README.md

```diff
@@ -3,29 +3,40 @@
 [![Forks](https://img.shields.io/github/forks/Airblader/i3.svg)](https://github.com/Airblader/i3/network)
 [![Stars](https://img.shields.io/github/stars/Airblader/i3.svg)](https://github.com/Airblader/i3/stargazers)
 
-# i3-gaps
+# Rounded i3-gaps
 
 ## What is i3-gaps?
```

Comment:
NIT: Maybe this title should be reworked since the section mentions both my fork and airbladers? Could be two seperate sections? Not a big deal either way